web/src/features/fba/pages/FireBehaviourAdvisoryPage.tsx

```diff
@@ -211,13 +245,15 @@ const FireBehaviourAdvisoryPage: React.FunctionComponent = () => {
         <Grid className={classes.flex} container direction={'row'}>
           <Grid item>
             <ZoneSummaryPanel
+              ref={sidePanelRef}
               selectedFireZone={selectedFireZone}
               fuelTypeInfo={hfiThresholdsFuelTypes}
               hfiElevationInfo={fireZoneElevationInfo}
               fireZoneAreas={fireZoneAreas}
+              className={classes.scrollablePanel}
             />
           </Grid>
-          <Grid className={classes.flex} item>
+          <Grid className={classes.flex} ref={mapRef} item>
```

Comment:
Continuing on from my comment above about an alternate approach to setting the height of the component.

```
<Grid
  className={classes.flex}
  sx={{height: `calc(100vh - ${formControlHeight + navRefHeight}px)`}}
  item
>
```